ci-operator/config/openshift-online/rosa-e2e/openshift-online-rosa-e2e-main__periodics.yaml

```
@@ -102,6 +102,35 @@ tests:
+    env:
+      CHANNEL_GROUP: nightly
+      CLUSTER_TOPOLOGY: classic
+      HOSTED_CP: "false"
+      LABEL_FILTER: "Platform:Classic"
+      OCM_LOGIN_ENV: staging
+      OPENSHIFT_VERSION: "4.23"
+      REPLICAS: "2"
+      STS: "true"
+    workflow: rosa-e2e-classic
+zz_generated_metadata:
 zz_generated_metadata:
```

YAMLlint (1.38.0)

[error] duplication of key "zz_generated_metadata" in mapping

(key-duplicates)